duckduckgo/iOS
# roll out home row experiment 2

Task/Issue URL: https://app.asana.com/0/414235014887631/751512552599839
Tech Design URL:
CC:

**Description**:

Removes experiment 1 and rolls out experiment 2 to everyone.

**Steps to test this PR**:

Clean install
1. CTA is shown after launch
1. No variant is on the atb any more 

Regression
1. Click "show me" shows the add to dock instructions
1. Click "no thanks" hides the CTA (and never see it again)

Instructions play button
1. Launch the instructions
1. When the video stops a play button appears
1. Tap the video to restart it


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
